py_bind/optv/parameters.pyx

```diff
@@ -0,0 +1,82 @@
+#Implementation of Python binding to parameters.h
+from libc.stdlib cimport malloc, free
+
+cdef class MultimediaParams:
+
+    def __init__(self, **kwargs):
+        
+        self._mm_np = <mm_np *>malloc(sizeof(mm_np))
+        
+        self.set_nlay(kwargs['nlay'])
+        self.set_n1(kwargs['n1'])
+        self.set_n2(kwargs['n2'])
+        self.set_d(kwargs['d'])
+        self.set_n3(kwargs['n3'])
+        self.set_lut(kwargs['lut'])
+    
+    def get_nlay(self):
+        return self._mm_np[0].nlay
+    
+    def set_nlay(self, nlay):
+        self._mm_np[0].nlay = nlay
+        
+    def get_n1(self):
+        return self._mm_np[0].n1
+    
+    def set_n1(self, n1):
+        self._mm_np[0].n1 = n1
+        
+    def get_n2(self):
+        list_size = sizeof(self._mm_np[0].n2) / sizeof(self._mm_np[0].n2[0])
+        n2_list = [None] * list_size
```

Comment:
Should use a numpy array, not a list. Many actions which require a loop for a list have an inbuild invisible loop in a numpy array.